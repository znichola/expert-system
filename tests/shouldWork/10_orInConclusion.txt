A => L | U
L => B
U => B
B => M
=A
?MLU