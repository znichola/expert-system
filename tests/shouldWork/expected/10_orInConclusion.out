M is True
L is Undetermined
U is Undetermined
